### ISSUE TITLE
NVDA does not support text review commands in controls in Visual Studio Code.

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#16246]
I've prepared a PR to try and address this issue.
### Steps to reproduce:
Most controls in Visual Studio Code do not support text review commands.
1. Open the directory containing the file in VS Code.
2. Press Ctrl+Shift+E to focus the file tree.
3. Try using the Numpad1/Numpad3 text review commands to review file names character by character in the file tree.

### Actual behavior:
NVDA only reports the character '0xfffc'.

### Expected behavior:
NVDA should be able to report filenames character by character.

#### NVDA installed/portable/running from source:
installed, portable, running from source:
#### NVDA version:
All Version
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.4046
#### Name and version of other software in use when reproducing the issue:
Visual Studio Code 1.86.2 
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
All versions are the same
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by brunoprietog** [comment]
Actually you can do it, just enter the child object and then go to the sibling.

**Comment by lukaszgo1** [comment]
This issue should be closed as a duplicate of #15159. The problem occurs not only in VS Code, so I don't think the proposed pull request is a correct solution.

**Comment by cary-rowen** [comment]
Hi @lukaszgo1 

Thanks for clarifying this.
There is already an app module for Visual Studio Code, and I thought #16248 would be appropriate as a solution to this particular problem, especially since #15159 has yet to be fixed.

Thanks

**Comment by lukaszgo1** [comment]
Note that this will not work for the web version of VS Code for example. Have you considered looking into #15159  if you are affected by it? Assuming your Pr is accepted, it should have a code comment explaining that the problem is caused by #15159 and that this solution should be reverted as soon as it gets fixed.
